S2S Integration Samples/README.md

```diff
@@ -2,7 +2,7 @@
 
 The directory contains samples for Service-to-service integration with Universal Print. This sample is provided to demonstrate how developers can use the [Universal Print Microsoft Graph API](https://aka.ms/UPGraphDocs) to integrate applications with Universal Print.
 
-A video walkthrough of this sample is [here](). The sample demonstrates how services can implement custom print functionality by hooking into and extending the print workflow. Build and run the sample to see how easy it is to update a print job's configuration to force all documents to print in black and white to save costs.
+A video walkthrough of this sample is [here](https://aka.ms/UPDevResources). The sample demonstrates how services can implement custom print functionality by hooking into and extending the print workflow. Build and run the sample to see how easy it is to update a print job's configuration to force all documents to print in grayscale to save costs.
```

Comment:
Suggest:
The code sample demonstrates how easily the cloud applications can register printers and create workflows to manage the jobs on a print queue as it transits Universal Print in Azure. 

Build and run this sample to intercept all jobs for a newly registered Universal Print printer, get notified when a new jobs arrives and then execute a simple task (force document to print in grayscale only).